onert-micro/onert-micro/include/core/OMRuntimeContext.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ONERT_MICRO_CORE_RUNTIME_CONTEXT_H
+#define ONERT_MICRO_CORE_RUNTIME_CONTEXT_H
+
+#include "OMStatus.h"
+
+#include "reader/OMCircleReader.h"
+
+#include <cstdint>
+
+namespace onert_micro
+{
+namespace core
+{
+
+class OMRuntimeContext
+{
+private:
+  reader::OMCircleReader _reader;
+
+public:
+  OMRuntimeContext() = default;
+  OMRuntimeContext(const OMRuntimeContext &) = delete;
+  OMRuntimeContext &operator=(const OMRuntimeContext &) = delete;
+  OMRuntimeContext &&operator=(const OMRuntimeContext &&) = delete;
+  OMRuntimeContext(OMRuntimeContext &&) = default;
```

Comment:
(Out of curiosity)
Move operator is deleted. but move constructor is not. Is this what you intended?

Reply:
Thank you for your review! Actually, yes - I need move property for `OMRuntimeContext` in one place of the code (in import part of the onert-micro) and in order to save binary I prefer delete move operator)

Reply:
I see. Thanks for your kindly reply.